packages/documentation/copy/en/reference/Namespaces and Modules.md

```diff
@@ -75,7 +75,7 @@ If you're converting a program from namespaces to modules, it can be easy to end
 - `shapes.ts`
 
   ```ts
-  export namespace Shapes {
+  declare module Shapes {
```

Comment:
Later in this section, `Shapes` is referenced as a module.
![obraz](https://user-images.githubusercontent.com/21079319/128888478-d314c908-ef07-484b-a016-2bd4a3e66135.png)